dark/index.html

```diff
@@ -307,6 +308,6 @@ <h3>CONTACT <span class="text-info">ME</span></h3>
       sidebar.classList.toggle("is-active");
     });
   </script>
-</body>
+</body>OOA
 
-</html>
+6[6~[[6~OOB</html>

```

Comment:
Can you tell me why did you add these changes? @maddy020

Reply:
I think It is a system error 
So sorry about that I have changed the file